mautic/mautic
# The display of flash messages is more flexible

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No]
| New feature/enhancement? (use the a.x branch)      | [Yes]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The display of flash messages is more flexible. CSS styles of the flash message depend on the type of message.

E.g., warning type flash message:
the previous one
![image](https://user-images.githubusercontent.com/96085911/152034389-67dad7b1-085e-4f8e-b72b-ce523fd45ef8.png) 
the new one
![image](https://user-images.githubusercontent.com/96085911/152034300-ef0fcb0e-9c21-4719-8a7e-3e8665b37a9e.png)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Modify and save something
3. Look at the flash message




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10819"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
